utils/config/examples/daos_server_mdonssd.yml

```diff
@@ -1,7 +1,7 @@
 # Example configuration file for Metadata on SSD.
 
 name: daos_server           # sys group daos_server
-access_points: ['example']  # management service leader (bootstrap)
+mgmt_svc_replicas: ['example']  # management service leader (bootstrap)
 # port: 10001               # control listen port, default 10001
```

Comment:
Seeing as this is an example, shouldn't we cleanup the comment string spacing?
```suggestion
name: daos_server               # sys group daos_server
mgmt_svc_replicas: ['example']  # management service leader (bootstrap)
# port: 10001                   # control listen port, default 10001
```
Same comment applies to other `utils/config/examples/daos_server_*.yml` files

Reply:
Sure, I can do that... Given how big the PR was already, though, I was trying to minimize changes that aren't directly related to the rename.

Reply:
Done.